frontend/components/LspAssets.tsx

```diff
@@ -137,28 +138,6 @@ export default function LspAssets() {
             ))
           : emptyLS7PAssets()}
       </Box>
-      <Box>
```

Comment:
we want this

Reply:
Okay I reverted, but if we do get it before submission we can always add it
are we going to submit it like this if it's not done?

Reply:
I think worth submitting with the placeholder for LSP8 if we dont finish. Products do that all the time to set user expectations and signal what's coming up